src/carExample/Car.java

```diff
@@ -14,13 +15,13 @@
     private List<CarWheels> wheels = new ArrayList<>();
     private List<CarDoors> doors = new ArrayList<>();
 
-    public Car(String productionDate) {
-        this.productionDate = productionDate;
+    public Car(LocalDate productionDate) {
+        this.productionDate = LocalDate.now();
     }
 
-    public Car(String productionDate, String engineType, int maxSpeed, double timeToHundreetKPH,
+    public Car(LocalDate productionDate, String engineType, int maxSpeed, double timeToHundreetKPH,
                int maxNumberOfPassengers, int actualNumberOfPassengers, int currentSpeed) {
-        this.productionDate = productionDate;
+        this.productionDate = LocalDate.now();
```

Comment:
you passed productionDate as constructor param but dont use it in constructor body

Reply:
added into main method